h5py_wrapper/wrapper.py

```diff
@@ -339,7 +339,9 @@ def _array_to_type(value, value_type):
                   'int': 'int',
                   'str': 'str',
                   'bool': 'bool',
-                  'Quantity': 'pq.Quantity'}
+                  'Quantity': 'pq.Quantity',
+                  'int64': 'np.int64',
+                  'float64': 'np.float64'}
```

Comment:
You're right. Let's just stick with 64 types for now. We should think about how we can support all numpy types without explicitly listing all of them, but that's beyond this PR.